src/particle/pdgid/functions.py

```diff
@@ -267,11 +278,13 @@ def is_sm_gauge_boson_or_higgs(pdgid):
     """
     Does this PDG ID correspond to a Standard Model gauge boson or Higgs?
 
-    Codes 21-30 are reserved for the Standard Model gauge bosons and the Higgs.
+    Codes 21-30 are reserved for the Standard Model gauge bosons and the Higgs,
+    but only the codes 21-25 actually correspond to SM particles.
     """
     if abspid(pdgid) == 24:  # W is the only SM gauge boson not its antiparticle
         return True
-    return True if pdgid in range(21, 31) else False
+
+    return True if pdgid in range(21, 26) else False
```

Comment:
```suggestion
    return True if 22 <= pdgid <=  26 else False
```

(You can use any combination of < and <=, depends on what is clearest)

Reply:
Keeping the range is perfectly fine too (here and below), on Python 3 it's just as fast.

Reply:
Happy to accept this same suggestion but 21 should be included ;-).

Reply:
Should be 21-25.

Reply:
I will accept the suggestion and will correct it. Then will push corrections for the CI.